lib/filterable/validators/valid_int_validator.rb

```diff
@@ -15,6 +15,6 @@ def is_integer_array?(value)
   end
 
   def is_integer_or_range?(value)
-    !!(/\A\d+(...\d+)?\z/ =~ value)
+    value.is_a?(Integer) || !!(/\A\d+(...\d+)?\z/ =~ value)
```

Comment:
I am a little confused by this. don't integer params usually come through as string representations of the integer? Like `priority=1` becomes `"1"` in the controller.

Reply:
@HParker I think you're right.

Regardless, you _should_ be able to accomplish what you're trying to do simply by adding `.to_s` to the old line 18... so:
`!!(/\A\d+(...\d+)?\z/ =~ value.to_s)`

Reply:
heres the circle build: https://circleci.com/gh/procore/procore/181974#tests/containers/6
it chokes on trying to convert an integer to a string. Could be that the test is setup improperly, who knows. But to be safe, think I'll add the `to_s` in there to keep it cleaner